src/components/network/network-table.js

```diff
@@ -380,6 +397,10 @@ const NetworkTable = (props) => {
         [rowFilter, selectedDataKey, formatCell]
     );
 
+    const isModifyingRow = useCallback(() => {
```

Comment:
Callback really necessary ?

Reply:
Yes it is. Without it, I have multiple warnings and lint do not pass.